client/my-sites/plans-features-main/components/free-free-dialog.tsx

```diff
@@ -0,0 +1,290 @@
+import { applyTestFiltersToPlansList, PLAN_PERSONAL } from '@automattic/calypso-products';
+import { Button, Dialog, Gridicon } from '@automattic/components';
+import { css, Global } from '@emotion/react';
+import styled from '@emotion/styled';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
```

Comment:
Nit: We can use `import type` for `TranslateResult`.

Reply:
I think not applicable anymore.